src/MeetingsSDKAdapter/controls/AudioControl.js

```diff
@@ -0,0 +1,62 @@
+import {Observable} from 'rxjs';
+import {map, distinctUntilChanged} from 'rxjs/operators';
+import {MeetingControlState} from '@webex/component-adapter-interfaces';
+import MeetingControl from './MeetingControl';
+
+/**
+ * Display options of a meeting control.
+ *
+ * @external MeetingControlDisplay
+ * @see {@link https://github.com/webex/component-adapter-interfaces/blob/master/src/MeetingsAdapter.js#L58}
+ */
+
+export default class AudioControl extends MeetingControl {
+  /**
+   * Attempts to mute the microphone of the given meeting ID.
+   * If the microphone is successfully muted, an audio mute event is dispatched.
```

Comment:
this is the jsdoc of the adapter handleLocalAudio

this one should read: calls the adapter handleLocalAudio() method